ui/ui-frontend/projects/referential/src/app/rule/rule.component.html

```diff
@@ -1,6 +1,11 @@
 <mat-sidenav-container [autosize]="true" [hasBackdrop]="false">
   <mat-sidenav #panel mode="side" position="end" [fixedInViewport]="true">
-    <app-rule-preview *ngIf="openedItem" (previewClose)="closePanel()" [rule]="openedItem"></app-rule-preview>
+    <app-rule-preview
+      *ngIf="openedItem"
+      (previewClose)="closePanel()"
+      [rule]="openedItem"
+      [readOnly]="(checkUpdateRole | async) === false"
```

Comment:
Il vaut mieux vérifier qu'il est différent de true (comme ça c'est en readonly si undefined)
```suggestion
      [readOnly]="(checkUpdateRole | async) !== true"
```